projects/kit/components/input-date-range/input-date-range.component.ts

```diff
@@ -157,6 +168,20 @@ export class TuiInputDateRangeComponent
         super(control, cdr, valueTransformer);
     }
 
+    @HostListener('click')
+    onClick(): void {
+        if (!this.isMobile && this.interactive) {
+            this.toggle();
+        }
+    }
```

Comment:
due to linter moved up from https://github.com/taiga-family/taiga-ui/pull/7932/files#diff-f8d088649bf22db138c442a2d8eb5d993a48f02cc6d3ec29cd7a30fb299b655aL272-L278